### PR TITLE
Missing URLs in POM

### PR DESCRIPTION
**Pull request by jccollet**
Missing URLs in POM

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,10 +15,12 @@
   <packaging>pom</packaging>
 
   <name>SonarTS</name>
+  <url>http://redirect.sonarsource.com/plugins/typescript.html</url>
```
Comment (vilchik-elena) [inline pom.xml]: @jccollet we have this url already in plugin pom

**Comment by vilchik-elena** [comment]
@jccollet are we clear that these changes will be in next release only?:)

**Comment by jccollet** [comment]
@vilchik-elena Yea, that was clear. I can live with that, it's not a big deal.

File: pom.xml (hunk 1 above)
Comment (jccollet) [inline pom.xml]: I know, but all plugins do include that URL in the top pom as well, so I'd rather play it safe.

**Pull request merged**